test/unit/spread.spec.ts

```diff
@@ -5,6 +5,7 @@ import { formatCode } from "../util";
 // TODO: Make some utils for testing other targets
 const expectUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toMatch(/[^.]unpack\(/);
 const expectTableUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toContain("table.unpack");
+const expectTsUnpack: util.TapCallback = builder => expect(builder.getMainLuaCodeChunk()).toContain("__TS__Unpack");
```

Comment:
Since we changed `expectTsMathAtan2` to `expectLualibMathAtan2`, I would suggest also changing this one to `expectLualibUnpack`